functions/Get-DbaAgentJobCategory.ps1

```diff
@@ -1,88 +1,81 @@
+            Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
 
-.PARAMETER Confirm
-Prompts you for confirmation before executing any changing operations within the command.
+        .NOTES
+            Author: Sander Stad (@sqlstad, sqlstad.nl)
+            Tags: Agent, Job, Job Category
 
-.PARAMETER EnableException
-        By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
-        This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
-        Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+            Website: https://dbatools.io
+            Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
+            License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
 
-.NOTES
-Author: Sander Stad (@sqlstad, sqlstad.nl)
-Tags: Agent, Job, Job Category
+        .LINK
+            https://dbatools.io/Get-DbaAgentJobCategory
 
-Website: https://dbatools.io
-Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
-License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
+        .EXAMPLE
+            Get-DbaAgentJobCategory -SqlInstance sql1
 
-.LINK
-https://dbatools.io/Get-DbaAgentJobCategory
+            Return all the job categories.
 
-.EXAMPLE
-Get-DbaAgentJobCategory -SqlInstance sql1
+        .EXAMPLE
+            Get-DbaAgentJobCategory -SqlInstance sql1 -Category 'Log Shipping'
 
-Return all the job categories.
+            Return all the job categories that have the name 'Log Shipping'.
 
-.EXAMPLE
-Get-DbaAgentJobCategory -SqlInstance sql1 -Category 'Log Shipping'
+        .EXAMPLE
+            Get-DbaAgentJobCategory -SqlInstance sstad-pc -CategoryType MultiServerJob
 
-Return all the job categories that have the name 'Log Shipping'.
+            Return all the job categories that have a type MultiServerJob.
 
-.EXAMPLE
-Get-DbaAgentJobCategory -SqlInstance sstad-pc -CategoryType MultiServerJob
-
-Return all the job categories that have a type MultiServerJob.
-
-#>
+    #>
     [CmdletBinding(DefaultParameterSetName = "Default")]
     param (
         [parameter(Mandatory = $true, ValueFromPipeline = $true)]
         [Alias("ServerInstance", "SqlServer")]
-        [DbaInstanceParameter[]]$SqlInstance,
+        [DbaInstance[]]$SqlInstance,
+        [Alias("Credential")]
         [PSCredential]$SqlCredential,
-        [Parameter(Mandatory = $false)]
         [ValidateNotNullOrEmpty()]
         [string[]]$Category,
         [ValidateSet("LocalJob", "MultiServerJob", "None")]
         [string]$CategoryType,
         [switch]$Force,
-        [switch][Alias('Silent')]$EnableException
+        [switch][Alias('Silent')]
+        $EnableException
```

Comment:
This one gets formated as such:

```
[Alias('Silent')]
[switch]$EnableException
```

Reply:
Created PR #3252 to fix this for all Get-DbaAgent... cmdlets. Thanks!